index.html

```diff
@@ -0,0 +1,63 @@
+<!-- Milestone 1, John Ieng -->
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="../styles.css" />
```

Comment:
`../styles.css` should be `./styles.css`. ".." points to the parent directory and "." points to the current direct; since index.html and styles.css are in the same directory, you will want to use "."